wiki/People/osu!monthly_Team/en.md

```diff
@@ -0,0 +1,30 @@
+# osu!monthly Team
+
+The contributors of [osu!monthly](/wiki/Community/osu!monthly) are often motivated and active players in the community, keeping track of what is going on each month to allow the project to live and perpetuate as the iterations go on.
+
+## Team members
+
+### Main contributors
+
+The people listed below are the core of the osu!monthly Team and participate every month in the production of the newspost, mainly by writing the articles.
+
+| Name | Role |
+| :-- | :-- |
+| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | Project lead |
+| ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) | Project editor |
+| ::{ flag=CA }:: [ianpep](https://osu.ppy.sh/users/7699030) | osu!taiko representative |
+| ::{ flag=ES }:: [Patatitta](https://osu.ppy.sh/users/16294034) | osu!catch representative |
+| ::{ flag=FR }:: [Pisapou](https://osu.ppy.sh/users/16640021) | Project editor |
+| ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) | Helper |
```

Comment:
mainly contributes with both mania plays and mania related news so I think this should still be noted. Subjective though
```suggestion
| ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) | Helper & osu!mania representative |
```